Marsiple/App/Post Details/PostDetailsView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  PostDetailsView.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class PostDetailsView: UIView {
+    private let postView = TitleAndBodyView.autolayoutView()
+    let tableView = UITableView.autolayoutView()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension PostDetailsView {
+    func updatePostProperties(title: String, body: String) {
```

Comment:
One way to avoid needing to write two basically same methods (`updatePostProperties` just invokes the `postView`'s method) is to expose the `postView`, so that it's accessible from the outside and it's `updateProperties` method can be invoked directly.

Reply:
Is that a good solution? Now someone can change other `postView` properties.

Reply:
There's a thin line to be walked here, but you're right - lets keep this well encapsulated.